server/src/main/java/dev/keva/server/command/impl/key/Incr.java

```diff
@@ -0,0 +1,47 @@
+package dev.keva.server.command.impl.key;
+
+import dev.keva.ioc.annotation.Autowired;
+import dev.keva.ioc.annotation.Component;
+import dev.keva.protocol.resp.reply.IntegerReply;
+import dev.keva.server.command.annotation.CommandImpl;
+import dev.keva.server.command.annotation.Execute;
+import dev.keva.server.command.annotation.ParamLength;
+import dev.keva.server.exception.CommandException;
+import dev.keva.store.KevaDatabase;
+
+import java.nio.charset.StandardCharsets;
+
+import static dev.keva.server.command.annotation.ParamLength.Type.EXACT;
+
+@Component
+@CommandImpl("incr")
+@ParamLength(type = EXACT, value = 1)
+public class Incr {
+    private final KevaDatabase database;
+
+    @Autowired
+    public Incr(KevaDatabase database) {
+        this.database = database;
+    }
+
+    @Execute
+    public IntegerReply execute(byte[] key) {
+        var afterIncr = 0L;
+        database.getLock().lock();
+        try {
+            byte[] valBytes = database.get(key);
+            long curVal = 0L;
+            if (valBytes != null) {
+                curVal = Long.parseLong(new String(valBytes, StandardCharsets.UTF_8));
```

Comment:
Not efficient, please refer [https://stackoverflow.com/questions/7619058/convert-a-byte-array-to-integer-in-java-and-vice-versa](https://stackoverflow.com/questions/7619058/convert-a-byte-array-to-integer-in-java-and-vice-versa)